pom.xml

```diff
@@ -117,6 +117,13 @@
             <version>6.1.0-TF</version>
         </dependency>
 
+        <dependency>
```

Comment:
[ERROR] Failed to execute goal on project totalfreedom: Could not resolve dependencies for project me.totalfreedom:totalfreedom:jar:5.0.2: Failure to find net.dv8tion:JDA:jar:3.8.0_423 in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]